app/containers/title-field.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+import { connect } from 'react-redux'
+import TitleField from '../components/title-field'
+import {
+  updateTitle,
+  activateTitleEditing,
+  editTitle,
+  deactivateTitleEditing
+} from '../actions'
+
+const makeMapStateToProps = (initialState, initialProps) => {
+  const { key } = initialProps.dat
+  const mapStateToProps = state => ({
+    dat: state.dats[key],
+    titleUnderEdit: state.titleUnderEdit
+  })
+
+  return mapStateToProps
+}
+
+const mapDispatchToProps = dispatch => ({
+  updateTitle: (key, path, editValue) =>
+    dispatch(updateTitle(key, path, editValue)),
+  activateTitleEditing: (title, editable) =>
+    dispatch(activateTitleEditing(title)),
+  editTitle: title => dispatch(editTitle(title)),
```

Comment:
I was confused by the name `editTitle`. Maybe `directTitleInput` or so? Tbh. `editTitle` seems like the only action that doesn't make sense while the others seem to fit perfectly.

Reply:
changed the name to `updateTemporaryTitleValue`